src/analyses/callstringAnalysis.ml

```diff
@@ -0,0 +1,87 @@
+open Analyses
+open GoblintCil
+open GobConfig
+
+(* Specifies the type of the callstack elements for the callstring analysis*)
+module type Callstack_Type =
+sig
+  include CilType.S
+  val stackTypeName: string
+  val pushElem: fundec -> ('d,'g,'c,'v) ctx -> t option (* returns an elements that should be pushed to the Callstack *)
```

Comment:
```suggestion
  val pushElem: fundec -> ('d,'g,'c,'v) ctx -> t option (* returns an element that should be pushed to the Callstack *)
```

I'm not sure if `pushElem` is the best name for this... Maybe we should think if we have a better idea.

Reply:
Sure, maybe something like "newEle",  "newStackEle", "stackEle". Or do you have a suggestion?